crates/bws/src/main.rs

```diff
@@ -90,6 +108,26 @@ enum GetCommand {
     Secret { secret_id: Uuid },
 }
 
+#[derive(Subcommand, Debug)]
+enum CreateCommand {
+    Project { name: String },
+}
+
+#[derive(Subcommand, Debug)]
+enum EditCommand {
+    #[clap(group = ArgGroup::new("edit_field").required(true).multiple(true))]
+    Project {
+        project_id: String,
```

Comment:
If you change this type to be Uuid here directly, you won't need to manually do Uuid::parse_str afterwards

Reply:
Great point, updated!